### PR TITLE
IGNITE-10803:[ML] Add prototype LogReg loading from PMML format

### PR DESCRIPTION
**Pull request by zaleslaw**

### CODE REVIEW

File: examples/pom.xml [hunk 1]
```diff
@@ -122,6 +122,30 @@
             <version>${javassist.version}</version>
             <scope>test</scope>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/org.jpmml/pmml-evaluator -->
+        <dependency>
+            <groupId>org.jpmml</groupId>
+            <artifactId>pmml-evaluator</artifactId>
```
Comment (vruusmann) [inline examples/pom.xml]: Consider directly depending on `org.jpmml:pmml-model(:1.4.7)` and `org.jpmml:pmml-model-metro(:1.4.7)`. Also, the PMML stuff really doesn't need any of the listed Jackson dependencies.

Comment (zaleslaw) [reply examples/pom.xml]: @vruusmann Great, thank you for the comment.
But without Jackson dependencies I've got a problem with JavaDoc generation.

Comment (zaleslaw) [reply examples/pom.xml]: Warning:java: unknown enum constant com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.ANY
  reason: class file for com.fasterxml.jackson.annotation.JsonAutoDetect$Visibility not found

Warning:java: unknown enum constant com.fasterxml.jackson.annotation.JsonInclude.Include.NON_EMPTY
  reason: class file for com.fasterxml.jackson.annotation.JsonInclude$Include not found

Warning:java: unknown enum constant com.fasterxml.jackson.annotation.JsonTypeInfo.Id.NAME
  reason: class file for com.fasterxml.jackson.annotation.JsonTypeInfo$Id not found

For example

Comment (nizhikov) [reply examples/pom.xml]: Hello @zaleslaw

Please, clarify, why you add jackson dependencies to examples module?
It brokes Spark Data Frame examples.

Comment (zaleslaw) [reply examples/pom.xml]: @nizhikov Sorry for the delay, currently, did you fixed that or I should fix this moment? Yes, I test with disabled scala profile probably and this is my mistake and I've ready to fix it

**Comment by nizhikov** [comment]
Hello, @zaleslaw.

Thanks for the response!

Yes, I fixed that, already.

You can see my commit, if interested - https://github.com/apache/ignite/commit/678e4455246cd6d000e4bd874524355afdf12806

Also, It's not only your fault :)
There was some misconfiguration on TC, so tests you broke with this change weren't tested.

File: examples/pom.xml (hunk 1 above)
Comment (nizhikov) [reply examples/pom.xml]: Hello, @zaleslaw.

Thanks for the response!

Yes, I fixed that, already.

You can see my commit, if interested - 678e445

Also, It's not only your fault :)
There was some misconfiguration on TC, so tests you broke with this change weren't tested.

**Pull request closed without merging**